### ISSUE TITLE
Wrong Error Message Text When Attempting to Open DMN

### ISSUE THREAD

**Issue by philippfromme** [zeebe-io/zeebe-modeler#213]
__Describe the Bug__

Trying to open a DMN diagram results in an error message with the wrong text.

![image](https://user-images.githubusercontent.com/7633572/79441291-56d95000-7fd7-11ea-8110-d6e545680ffe.png)

__Steps to Reproduce__

1. Try opening a DMN diagram
2. Error message pops up

__Expected Behavior__

Error message reads "diagram.dmn" is not a BPMN file.

__Environment__

- Zeebe Modeler Version: 0.9.0

**Comment by imrishabh18** [comment]
@pinussilvestrus  Can I take up this issue?

**Comment by pinussilvestrus** [comment]
Of course! Feel free to open a PR and/or ask for help if you need some 🚀

**Comment by imrishabh18** [comment]
What should be the text for the error message then?

**Comment by pinussilvestrus** [comment]
There is a hint in the issue description

> Error message reads `"diagram.dmn" is not a BPMN file.`


Despite, you can always come up with another solution if you think there is something better.

**Comment by vvkpd** [comment]
Is this card is ready to pick, i would like to work upon.

**Comment by pinussilvestrus** [comment]
Feel free to open a PR. There was already work done in this regard: https://github.com/zeebe-io/zeebe-modeler/pull/256, which was not finished.
